src/platform/common/configSettings.ts

```diff
@@ -90,6 +90,8 @@ export class JupyterSettings implements IWatchableJupyterSettings {
     // Hidden settings not surfaced in package.json
     public disableZMQSupport: boolean = false;
     // Hidden settings not surfaced in package.json
+    public useJupyterDebugger: boolean = false;
```

Comment:
Just like zmq have a hidden setting that will enable use to use Jupyter protocol debugger for local kernels as well.
Good for testing.